kfserving/kfserving-install/base/config-map.yaml

```diff
@@ -89,8 +89,8 @@ data:
     }
   ingress: |-
     {
-        "ingressGateway" : "knative-ingress-gateway.knative-serving",
-        "ingressService" : "kfserving-ingressgateway.istio-system.svc.cluster.local"
+        "ingressGateway" : "kubeflow-gateway.kubeflow",
```

Comment:
Why is this ingressgateway called the kubeflow-gateway.kubeflow?

Isn't the istio gateway the only one we need?

Reply:
kubeflow uses this`kubeflow-gateway` in `kubeflow` namespace. istio-ingress gateway is not being used as it's in `istio-system`.

This PR reverts some changes in https://github.com/kubeflow/manifests/pull/949 and bump the version.